routes/index.js

```diff
@@ -27,6 +27,7 @@ module.exports = function (express) {
   router.use('/user', require('./users')(express))
   router.use('/post', require('./posts')(express))
   router.use('/feed', require('./feed')(express))
+  router.use('/search', require('./search')(express))
```

Comment:
Wouldn't it make more sense to extend `/api/users`? We are searching users here, right?